SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agents-general.html

```diff
@@ -83,7 +83,6 @@
           class="wz-tooltip">
           Start time: {{ agentInfo.syscheck.start || 'Unknown'}}
           <br> End time: {{ agentInfo.syscheck.end || 'Unknown'}}
-          <br> Duration time: {{ agentInfo.syscheck.duration +' minutes' || 'Unknown'}}

```

Comment:
Why you deleted this? Take a look at the Kibana app, you must calculate it and then show it